plugins/@grouparoo/zendesk/__tests__/integration/destination-mapping-options.ts

```diff
@@ -1,13 +1,10 @@
-import path from "path";
```

Comment:
can you undo the changes to this file (and all the zendesk ones)? I feel like you have something removing unused things or something.
that gets weird when we do this comment in and out the nock thing.